### ISSUE TITLE
Changelog on PR's since last tag?

### ISSUE THREAD

**Issue by mtirion** [mikepenz/release-changelog-builder-action#650]
Sorry, more of a question but ... I have release workflow that is triggered manually on the main branch. In that workflow I have these steps: 1) checkout 2) install GitVersion 3) build and package in zip-file 4) generate changelog 5) create tag&release and publish.

The problem that I have is that the tag is created in step 5, where step 4 is taking the PR's from tag n and n-1. Is there a way to have the action get the PR's from the last tag until 'now' (so, without a new tag)? Or what would be a better way to solve this?

**Comment by mikepenz** [comment]
@mtirion you could provide the Git SHA-1 to the `toTag` config, and the previous tag as `fromTag` so the action won't try to identify these based on the found tags on the repo. 

-----

Would it be a possibility to place the git tag before step 4 and do the release afterwards?

**Comment by mtirion** [comment]
Thanks for the quick reply @mikepenz! I will try that, makes sense. For creating the tag and release and upload I use the GitHub Action ncipollo/release-action@v1 that does it all. I've tried to change it to separate steps, and than rearrange the steps, but I ran into all kinds of other issues. I'll try your solution first ... and will post the result here for others to learn :)

**Comment by mikepenz** [comment]
This action itself for example does it like this: https://github.com/mikepenz/release-changelog-builder-action/blob/develop/.github/workflows/ci.yml#L105-L119

(Tag being the trigger for the release procedure to run)

**Comment by mtirion** [comment]
Here is what I did now (which works). I've set the fromTag to ${{ steps.gitversion.outputs.VersionSourceSha }} and toTag to ${{ steps.gitversion.outputs.Sha }}. These are the variables provided by GitVersion. This gives me the correct result in the CHANGELOG.

The reason that I didn't chose for tag-creation to be the trigger, is that I want the versions to be done using GitVersion and not manually. This way I can also create a tag for bumping up the major version for instance without immediately publishing the release for it.

Thanks for thinking about another option though. Much appreciated.
